### DISCUSSION TITLE
Does whisper only support traditional Chinese？

### DISCUSSION THREAD

**Discussion by beiweixiaowang** [openai/whisper#987] [Q&A] [answered]
I found that the result of Chinese recognition is traditional. Do you support simplified Chinese?

**Comment by bangyuanzhu** [comment]
It may not support it, Try to use Opencc package to transfer the text.
`import opencc`
`converter = opencc.OpenCC('t2s.json')`
`simplified_txt = converter.convert(txt)`

**Reply by thredreams** [reply]
Or you can try zhconv
```
from zhconv import convert
transcribe = whisper. Decode(model, mel, options)
simplified_txt = convert(transcribe.text, 'zh-cn')
```

**Reply by beiweixiaowang** [reply]
Thanks to your reply, I found an interesting phenomenon, it may be
that different sizes of model training corpus are different,
whisper-base can output a part of the Chinese Simplified. I have
solved the problem by using zhconv.
```
def audio_to_text(model, audio_path):
    result = model.transcribe(audio_path)
    return zhconv.convert(result['text'], 'zh-cn')
```

**Comment by jongwook** [comment, marked as answer]
It may transcribe in either form since it was trained on both. Using a converter library as others suggested is a simple way to fix it, and you can also try the prompting method as shown in https://github.com/openai/whisper/discussions/277#discussioncomment-3832154.

**Comment by beiweixiaowang** [comment]
Thanks, I'll check it out.

Jong Wook Kim ***@***.***> 于2023年3月6日周一 16:46写道：
